scripts/e2e_test_tosa.sh

```diff
@@ -13,27 +13,25 @@
 
 set -e
 
-if [[ $# -ne 8 ]] ; then
-  echo "Usage: $0 <path/to/model> <path/to/emitc/reference-implementation/include/> <path/to/tf-opt> <path/to/emitc-opt> <compiler> <batch-size> <seed> <output_dir>"
+if [[ $# -ne 7 ]] ; then
+  echo "Usage: $0 <path/to/model> <path/to/emitc/reference-implementation/include> <path/to/emitc-opt> <compiler> <batch-size> <seed> <output_dir>"
   echo
   echo "Both a keras and a tensorflow saved model is supported."
   echo
   echo "This script expects a python version in the PATH with a recent version of tensorflow installed."
-  echo "You need to compile tensorflow from source to build the tf-opt program."
-  echo "Tested with python 3.6.9 and tensorflow 2.4.0"
+  echo "Tested with python 3.8.10 and tensorflow 2.9.0-dev20220121"
```

Comment:
```suggestion
  echo "Tested with python 3.8.10 and tf-nightly 2.9.0-dev20220121"
```

Reply:
```suggestion
  echo "Tested with python 3.8.10 and tf-nightly 2.8.0.dev20211102"
```

Also works like charm with the version we specify in our `requirements.txt`. Thus, there is no need to wait on #254 if we adjust it here accordingly.